en/admin/migration.inc

```diff
@@ -182,6 +190,18 @@ The following table shows [options] available with the **cubrid loaddb** utility
 
         cubrid loaddb -p admin -d demodb_objects newdb
 
+.. option:: -S, --SA-MODE
+
+    This option loads data into a database in standalone mode, without the necessity of a running server. This is the default loading mode of **loaddb**. ::
+
+        cubrid loaddb -S -u admin -p admin -d demodb_objects newdb
+
+.. option:: -C, --CS-MODE
+
+    This option requires a server process running on the database in which the data is being loaded. Unlike the standalone mode of **loaddb**, the client-server mode will load the data using multiple threads on the server, meaning greater performance for large amounts of data. ::
```

Comment:
```suggestion
    This option connect to a running server process. Unlike the standalone mode of **loaddb**, the client-server mode loads the data using multiple threads leading to greater performance for large amounts of data. Multiple **loaddb** session may run at the same time. ::
```

Reply:
Repeat restriction here, cannot load object references and shared/class attributes. Are there other restrictions?